packages/cli/src/commands/generate/scaffold/templates/components/Name.tsx.template

```diff
@@ -23,15 +28,15 @@ const formatEnum = (values: string | string[] | null | undefined) => {
   }
 }
 
-const jsonDisplay = (obj) => {
+const jsonDisplay = (obj: unknown) => {
   return (
     <pre>
       <code>{JSON.stringify(obj, null, 2)}</code>
     </pre>
   )
 }
 
-const timeTag = (datetime) => {
+const timeTag = (datetime?: string) => {
```

Comment:
So, out of Prisma and the query, the date time value is a string and not a proper date?

Right, the ISO JSON value. This type here definitely helps to recognize that it's a string and not a date.